django_north/management/commands/migrate.py

```diff
@@ -77,21 +78,44 @@ def migrate(self):
     def _load_schema_file(self, file_name, load_name=None):
         file_path = os.path.join(
             settings.NORTH_MIGRATIONS_ROOT, 'schemas', file_name)
+        self._load_schema_path(file_path, load_name or file_name)
+
+    def _load_schema_path(self, file_path, load_name):
         if self.verbosity >= 1:
-            if load_name is None:
-                load_name = file_name
             self.stdout.write(
                 self.style.MIGRATE_LABEL("Load {}".format(load_name)))
         self.run_script(file_path)
 
+    def _load_schema_fullpath_file(self, file_path):
+        migration_root = os.path.join(
+            settings.NORTH_MIGRATIONS_ROOT, 'schemas')
+        load_name = os.path.relpath(file_path, migration_root)
+        self._load_schema_file(file_path, load_name)
+
+    def _load_schema_string(self, input_string):
+        path = os.path.join(
+            settings.NORTH_MIGRATIONS_ROOT, 'schemas', input_string)
+
+        if os.path.isdir(path):
+            for file_name in sorted(os.listdir(path)):
+                file_path = os.path.join(path, file_name)
+                if os.path.isfile(file_path) and file_path.endswith('.sql'):
+                    self._load_schema_fullpath_file(file_path)
+        elif os.path.isfile(path):
+            self._load_schema_fullpath_file(path)
+        else:
+            for file_path in sorted(glob.glob(path)):
+                if os.path.isfile(file_path) and file_path.endswith('.sql'):
```

Comment:
But what if the glob points to a dir, though ?

Reply:
I don't think it's a expected behavior to load multiple files and dird over a glob. And we don't want a recursive search.

I don't know use case to include a glob of dirs, and I don't think it was desired.

But I can do it if you think it's better ?

Reply:
I don't know. I'd find it logical if it did, but I really don't know if it's ever going to be used. For me it's more along the lines of "it it simplifies the code, let's do it".